backend/core/endpoints/channels.py

```diff
@@ -64,11 +64,9 @@ def create_channel(
         if server is None:
             raise HTTPException(status_code=404, detail="Server not found")
         channel = schemas.ChannelCreate(
-            **{
-                "discordId": channel.discord_id,
-                "name": channel_data.get("name"),
```

Comment:
Probably should discuss if we want to change this everywhere or leave it the old way

Reply:
Yea, we probably should change them all into this, there is no real reason to keep it the old way

Reply:
Made the rest of the changes in acb143eeae13faa9106399177b0515a4e70c8e5a, now branch is ready to merge imo